release(int-6026): 2.0.1 (#31)
DataDome/datadome-flutter: CHANGELOG.md

## 2.0.1 (2025-11-19)

> * Fix: On iOS, POST HTTP request with a `FlutterStandardTypedData` body are no longer emitted with an empty body

Reviewer: nitpick:
This seems to be a new format. 
For consistency, instead of using the prefix `Fix:` , could you rephrase it as `Fix ...` in a complete sentence like other entries in this changelog?

Author: I'll open an ADR to propose this new format. It is so much easier to make sentence like this.

I'll try to make a "full sentence" then

Reviewer: I totally agree.

Author: ```suggestion
* Fix on issue on iOS where POST HTTP requests could be sent without the expected body when giving a `FlutterStandardTypedData` as the body.
```

> * Fix: On iOS, POST HTTP request with a `FlutterStandardTypedData` body are no longer emitted with an empty body

Reviewer 2: ```suggestion
* Fix `FlutterStandardTypedData` request body parsing for POST HTTP body on iOS
```

> * Fix on issue on iOS where POST HTTP requests could be sent without the expected body when giving a `FlutterStandardTypedData` as the body.

Reviewer: ```suggestion
* Fix an issue on iOS where POST HTTP requests could be sent without the expected body when giving a `FlutterStandardTypedData` as the body.
```

> * Fix an issue on iOS where POST HTTP requests could be sent without the expected body when giving a `FlutterStandardTypedData` as the body.

Reviewer 3: nitpick: There's no punctuation on past entries, and it's specified in the guidelines.

```suggestion
* Fix an issue on iOS where POST HTTP requests could be sent without the expected body when giving a `FlutterStandardTypedData` as the body
```
